src/Widgets/PagesList.vala

```diff
@@ -279,8 +279,7 @@ public class ENotes.PagesList : Gtk.Box {
 
         listbox.row_activated.connect ((row) => {
             window.toggle_edit ();
-
-            if (ENotes.ViewEditStack.current_mode == Mode.EDIT) {
+            if (ENotes.ViewEditStack.current_mode == Mode.EDIT && settings.keep_sidebar_visible) {
                 ENotes.Editor.get_instance ().give_focus ();
```

Comment:
Editor should always gain focus if you move to the editor. You're moving to the editor for a reason aren't you? ;)